generated/nifake/tests/test_session.py

```diff
@@ -246,10 +291,111 @@ def test_array_input_function(self):
             session.array_input_function(test_array)
             self.patched_library.niFake_ArrayInputFunction.assert_called_once_with(SESSION_NUM_FOR_TEST, test_array_size, test_array)
 
-    # TODO(marcoskirsch): Other read variations: waveform with ViReal64 and ViInt16 * 3 mechanisms
-
-    # TODO(marcoskirsch):
-    # def test_multiple_outputs of different types
+    # TODO(marcoskirsch): Waveforms
```

Comment:
I think it's safe to remove this TODO because the generator doesn't yet support waveforms.
When/if we add it, we'll have to add it together with the test coverage.

Reply:
Fixed